src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const topUL = document.createElement('ul');
+
+  for (const topKey in data) {
```

Comment:
It might be beneficial to check if 'data' is an object before iterating over it. This can prevent potential runtime errors if 'data' is not an object.